### ISSUE TITLE
increased info on cryptcheck

### ISSUE THREAD

**Issue by ljr9600** [ncw/rclone#2630]
#### What is your current rclone version (output from `rclone version`)?
1.42

#### What problem are you are trying to solve?
cryptcheck

#### How do you think rclone should be changed to solve that?
I'd like to see more info on cryptcheck, especially for long/large checks.  I routinely check 2-3TB of data (200,000 files) and would like to see the bytes/mbytes/gbytes checked, as a good estimation of how long is left, since it seems impossilble to show the eta of completion time.

**Comment by ncw** [comment]
I guess you are looking at `--stats` output to see the progress.

`-P` works with cryptcheck too.

However the expensive calculate the hash part doesn't have progress as you've noted.  This is because rclone doesn't instrument the transfer as it knows it is a local transfer.

It would be relatively easy to get rclone to count the total stats so files/MB done, files/MB to go - would that help?

I think counting the transfers of individual files is more difficult.

**Comment by ljr9600** [comment]
yes, that would be great since as you said, rclone know how many local bytes processed.

**Comment by ljr9600** [comment]
also would be nice to see the mb/sec read time for the local drive crc check.
